filter_lists/regional.json

```diff
@@ -527,7 +527,7 @@
     {
         "uuid": "AD3E8454-F376-11E8-8EB2-F2801F1B9FD1",
         "url": "https://raw.githubusercontent.com/tcptomato/ROad-Block/master/road-block-filters-light.txt",
-        "title": "Romanian Ad (ROad) Block List Light",
+        "title": "Romanian Ad (ROad) Fliterlist",
```

Comment:
@antonok-edm — there's a typo here: "Fliterlist" should be "Filterlist"

Reply:
removed typo